src/structs/Item.js

```diff
@@ -295,6 +296,13 @@ export class Item extends AbstractStruct {
      * @type {ID | null}
      */
     this.redone = null
+    /**
+     * If this item was referenced by other weak links, here we keep the references
+     * to these weak refs.
+     * 
+     * @type {Set<YWeakLink<any>> | null}
+     */
+    this.linkedBy = null
```

Comment:
This will increase the size of all Item objects. This should be stored on AbstractType instead.